Sources/kai/AST.swift

```diff
@@ -76,8 +76,8 @@ indirect enum AstNode {
     case stmtIf(cond: AstNode, body: AstNode, AstNode?, SourceRange)
     case stmtReturn([AstNode], SourceRange)
     case stmtFor(initializer: AstNode?, cond: AstNode?, post: AstNode?, body: AstNode, SourceRange)
-    case stmtSwitch(cond: AstNode?, body: AstNode, defaultBody: AstNode?, SourceRange)
-    case stmtCase(cond: AstNode, body: AstNode, SourceRange)
+    case stmtSwitch(subject: AstNode?, cases: [AstNode], SourceRange)
+    case stmtCase(AstNode?, body: AstNode, isDefault: Bool, SourceRange)
```

Comment:
Shouldn't need both Optional match and an `isDefault` should we? Doesn't a nil match indicate it's the default?